lib/pages/chat/events/message_content.dart

```diff
@@ -502,10 +507,13 @@ class _MessageVideoBuilder extends StatelessWidget {
 
   final DownloadVideoEventCallback handleDownloadVideoEvent;
 
+  final ValueNotifier<Either<Failure, Success>> searchStatus;
```

Comment:
it is `stateless`. How about the initialize state for search? What happen if the chat not in `search`

Reply:
made it optional